frontend/src/pages/Profile/UserSkills/UserSkills.tsx

```diff
@@ -36,19 +36,28 @@ interface Props {
     }>;
   }>;
   setSearchSkill: (value: string) => void;
+  searchSkill: string;
 }
 
-const UserSkills: React.FC<Props> = ({ userSkills, setSearchSkill }) => (
+const UserSkills: React.FC<Props> = ({ userSkills, setSearchSkill, searchSkill }) => (
   <SkillsBox>
     <SearchWrapper>
       <SearchSkill
         disableUnderline
         placeholder="Search"
+        inputProps={{ maxLength: 100 }}
+        fullWidth
         startAdornment={
           <InputAdornment position="start">
             <SearchIcon color="disabled" />
           </InputAdornment>
         }
+        onKeyDown={(event) => {
+          const { key } = event;
+          if (key === ' ' && searchSkill.length === 0) {
+            event.preventDefault();
+          }
+        }}
```

Comment:
Move it please to the separate function!
We can't check for string with space. Replace this check with lodash(https://lodash.com/docs/4.17.15) or js trim method!

Reply:
Moved and changed check